src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/IChatClient.cs

```diff
@@ -33,6 +33,22 @@ public interface IChatClient : IDisposable
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>The response messages generated by the client.</returns>
     /// <exception cref="ArgumentNullException"><paramref name="messages"/> is <see langword="null"/>.</exception>
+    /// <remarks>
+    /// Prompt <paramref name="messages"/> are passed through to the underlying provider, and the <see cref="ChatResponse.Messages"/>
+    /// in the <see cref="ChatResponse"/> are returned without modification or filtering.
```

Comment:
I understand the intent, but I think this is misleading. One of the strengths of the middleware approach is exactly that IChatClient implementations in the pipeline _can_ manipulate these objects, in both directions. FunctionInvokingChatClient, for example, is augmenting the messages with its own. Someone could write a SummarizeHIstoryChatClient that replaced the whole contents with a summarized version that was then forwarded along. Etc.

Reply:
Good point; thanks. We could remove the opening sentence entirely, change "Callers are expected..." to "Applications are expected..." and the declarations to include "by default."